sdk/src/test/kotlin/io/rover/sdk/polls/VotingInteractorSpec.kt

```diff
@@ -52,14 +51,19 @@ object VotingInteractorSpec : Spek({
         whenever(optionResult.encodeJson()).thenReturn(jsonObject)
         whenever(jsonObject.toString()).thenReturn(optionResultResponse)
 
+        var optionYielded = false
+        votingInteractor.optionResults.subscribe {
+            optionYielded = true
+        }
+
         votingInteractor.fetchVotingResults(pollId, optionIds)
 
         it("sets poll results in storage") {
             verify(votingStorage).setPollResults(pollId, optionResultResponse)
         }
 
         it("updates option results subject") {
-            verify(optionResultsSubject).onNext(optionResult)
+            assert(optionYielded)
```

Comment:
Hm, do we know for sure that -ea is passed to the JVM when running the tests? Maybe better to use the assertions from spek like assertTrue/assertEquals.